android/festago/app/src/main/java/com/festago/festago/data/dto/MemberTicketResponse.kt

```diff
@@ -0,0 +1,36 @@
+package com.festago.festago.data.dto
+
+import com.festago.festago.domain.model.Ticket
+import com.festago.festago.domain.model.TicketState
+import kotlinx.serialization.Serializable
+import java.time.LocalDateTime
+
+@Serializable
+data class MemberTicketResponse(
+    val id: Long,
+    val number: Int,
+    val entryTime: String,
+    val state: String,
+    val reservedAt: String,
+    val stage: StageResponse,
+    val festival: MemberTicketFestivalResponse,
+) {
+    fun toDomain() = Ticket(
```

Comment:
```suggestion
    fun toDomain(): Ticket = Ticket(
```
반환 타입은 명시적으로 하는게 어떤가요??